config/samples/xgboost-dist/utils.py

```diff
@@ -289,7 +287,7 @@ def parse_parameters(input, splitter_between, splitter_in):
     for kv in ky_paris:
         conf = kv.split(splitter_in)
         key = conf[0].strip(" ")
-        if key == "objective":
+        if key == "endpoint":
```

Comment:
Why changing this line?

Reply:
I was not sure where objective was used and hence added it for endpoint. if objective is used for anycase, I can add one more if loop for endpoint.

Reply:
/cc @merlintang to double check

Reply:
@RudraprakashR please keep the object function, since it would be used by xgboost algorithm